requirements/dev.txt

```diff
@@ -19,9 +21,9 @@ ddt==1.2.1
 django-nose==1.4.6
 django-waffle==0.18.0
 django==1.11.26
-djangorestframework-jwt==1.11.0
```

Comment:
edx-platform currently lists this as an explicit dependency but doesn't seem to import code directly from it; when updating edx-platform to use the new release of edx-drf-extensions, please make sure to do any required work to remove this dependency there also.